packages/vue-app/template/client.js

```diff
@@ -611,7 +606,15 @@ function showNextPage (to) {
     layout = layout(app.context)
   }
   this.setLayout(layout)
-  <% } %>
+  next()
+}
+<% } %>
+
+function checkForErrors (to) {
```

Comment:
to argument is unused. Can we use (app) without binding this?

Reply:
Sure.

Reply:
Let me know if it's okay now.

Reply:
Looks like also we got a CI error but seems to be unrelated?